src/writeData/subscriptions/components/SubscriptionCard.tsx

```diff
@@ -22,6 +23,7 @@ interface Props {
 
 const SubscriptionCard: FC<Props> = ({subscription}) => {
   const {deleteSubscription} = useContext(SubscriptionListContext)
+  const timeSince = moment(subscription.updatedAt).fromNow()
```

Comment:
Instead of moment I think you can use `luxon` to calculate relative datetimes.

Reply:
good catch, done!